functional/keylime-policy-commands/test.sh

```diff
@@ -324,6 +330,30 @@ rlJournalStart
         popd ||:
     rlPhaseEnd
 
+    rlPhaseStartTest "Create measured boot policy"
+        mkdir -p measured-boot
+        pushd measured-boot || exit
```

Comment:
I don't expect this to fail but if it would, `exit` would mean that we won't do any clean up, which is not desired.

Reply:
Right, I'll change it to make the test fail